course_access_groups/utils.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
Can we use a different module name for this? One of my colleagues and I had discussions over time with using files called "util". Then encourage dumping functions that do not quite match other places. He convinced me that avoiding "utils" is a good practice for maintainability. 

And the function `add_course_fields` is not a utility method so should not be in a 'utils' module anyway, if we had an already existing `utils` module.

Instead, would you please call this module  `course_access_groups.monkeypatch` so readers are very clear this is workaround hacking?

Reply:
Great idea! Will do so. Thanks John!

Reply:
Thank you!